robottelo/ui/computeresource.py

```diff
@@ -63,6 +63,7 @@ def _assign_locator_value(self, target, value):
             pass
         else:
             self.page.assign_value(target, value)
+        self.page.wait_for_ajax()
```

Comment:
both `click` and `assign_value` do `wait_for_ajax` in the end of their execution, so why do we need it?

Reply:
assign_value do not do wait_for_ajax, and we need that to be performed

Reply:
sorry you right, the select and input instead does do  wait_for_ajax